libs/core/src/stories/Welcome.stories.mdx

```diff
@@ -3,14 +3,19 @@ import { Meta, Story } from '@storybook/addon-docs';
 <Meta title="Welcome" parameters={{ previewTabs: { canvas: { hidden: true }, } }} />
 
 # Building with Sage
-Our system is our source of truth, providing everything you need to build the best experiences for all Kajabi entrepreneurs.
+Our design system is the ultimate source of truth, offering a comprehensive and well-structured set of guidelines, components, and resources that empower designers and developers to create exceptional experiences for all Kajabi entrepreneurs. By adhering to our system, we can ensure a consistent, unified, and high-quality user experience across all touchpoints within the Kajabi platform.
 
 ## Foundations
-Our system starts with some core visual elements: Type, Color, and Icons. We also prescribe common practices for aspects of the system.
+Our design system is built upon a foundation of three core visual elements: Typography, Color, and Icons. These fundamental components work harmoniously to create a consistent and cohesive visual language throughout our designs.
 
 ## Components
-Components are the backbone of our design system. In order to be most usable, our components are written in both Rails and React form.
-Code examples, property options, and best practices can all be discovered here. Links to Figma design documentation can also be found in this section.
+Components are the backbone of our design system, acting as the building blocks for crafting seamless and cohesive user interfaces. Our components are available for web components and React, which maximizes usability and ensure compatibility with different development environments.
+
+Our extensive documentation provides code examples, property options, and best practices for each component, making it easy for designers and developers to understand and implement them effectively. This thorough resource empowers teams to create consistent, high-quality experiences while reducing the learning curve and boosting productivity.
 
 ## Contributing
-We encourage everyone to feel empowered to participate. Understanding and contributing to the system are the keys to the system's longevity.
+We firmly believe in fostering an inclusive and collaborative environment where everyone feels empowered to participate in the evolution of our design system. The success and longevity of the system depend on the collective efforts and insights of our diverse team, as well as the users who interact with it daily.
```

Comment:
>depend
Maybe? 
"depends"

Reply:
Thanks for the feedback! I ran this one through Grammarly, and it seems to agree that "depend" is correct, "the singular verb **depends** does not agree with the plural compound subject."

Reply:
Huh, reads wrong to me but 🤷‍♂️